domains/praveen.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "praveentcom",
+    "email": "shield@praveent.com"
+  },
+  "records": {
+    "A": ["216.198.79.1"],
+    "TXT": "vc-domain-verify=praveen.is-a.dev,260199b0ff20b680494d"
```

read and follow the step 5-> https://docs.is-a.dev/guides/vercel/

```suggestion
    "A": ["216.198.79.1"]
```